cypress/e2e/awx/resources/inventoryGroup.cy.ts

```diff
@@ -93,6 +93,7 @@ describe.skip('Inventory Groups', () => {
         //1) Use the inventory created in beforeEach block, access the groups tab of that inventory
         cy.navigateTo('awx', 'inventories');
         cy.filterTableBySingleSelect('name', inventory.name);
+        cy.wait(2000);
```

Comment:
Is there a way for us to intercept some of the API calls and wait for them instead of waiting X amount of time?